.github/workflows/maven.yml

```diff
@@ -22,3 +22,9 @@ jobs:
         java-version: 11
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+    - run: mkdir package && cp target/*.jar package
+    - uses: actions/upload-artifact@v2
+      with:
+        name: Package
+        path: package
+
```

Comment:
is this really needed on informatici repo? @ArnaudFonzam

Reply:
not sure

Reply:
maybe @npasquetto knows?